frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -307,6 +287,23 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({
     makeOCSRequest();
   };
 
+  const handleStorageClass = (sc: string, provisioner: string) => {
+    setStorageClass(sc);
+
+    if (_.isEqual(provisioner, 'kubernetes.io/no-provisioner')) {
```

Comment:
```suggestion
    if ( provisioner ===  'kubernetes.io/no-provisioner')) {
```
This is the second place I am seeing this being used. Consider putting `no-provisioner` in the consts file.